Goal: src/newgrp.c: don't call sanitize_env()
Fixes: #22244

Status wrt upstream: Upstream do not call sanitize_env.
                     As newgrp code is quite different in upstream, it could be
                     nice if Tomasz could check http//bugs.debian.org/22244
                     (IMO, upstream is correct)

Index: shadow-4.0.3/src/newgrp.c
===================================================================
--- shadow-4.0.3.orig/src/newgrp.c	2005-05-22 23:13:45.907167000 +0200
+++ shadow-4.0.3/src/newgrp.c	2005-05-22 23:16:36.157167000 +0200
@@ -91,7 +91,7 @@
 
 #if ENABLE_NLS
 	/* XXX - remove when gettext is safe to use in setuid programs */
-	sanitize_env ();
+	/* sanitize_env ();*/
 #endif
 
 	setlocale (LC_ALL, "");
